src/components/newContact/NewContact.tsx

```diff
@@ -3,94 +3,54 @@ import styles from './newContact.module.scss'
 import { useDispatch, useSelector } from 'react-redux'
 import { selectIsLoading, selectNewContact } from '../../redux/contacts/selectors'
 // import { Contact } from '../../models/interfaces/contact'
+import { setNewContact, saveNewContact } from '../../redux/contacts/actions'
+
 
 // Components
 import Header from '../header/Header'
-import { setNewContact } from '../../redux/contacts/actions'
-import { FormEvent } from 'hoist-non-react-statics/node_modules/@types/react'
+import { Contact } from '../../models/interfaces/contact'
 
 const NewContact: FC = (props) => {
   const loading = useSelector(selectIsLoading);
   const newContact = useSelector(selectNewContact);
 
   const dispatch = useDispatch();
-  // const [contactInfo, setContactInfo] = useState({
-  //   firstName: '',
-  //   lastName: '', 
-  //   phone: '',
-  //   email: '',
-  //   address1: '',
-  //   address2: '',
-  //   city: '',
-  //   state: '',
-  //   zip: ''
-  // })
-
-  // const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-  //   e.persist();
-  //   setContactInfo((prevState) => ({
-  //     ...prevState,
-  //     [e.target.name]: e.target.value,
-  //   }));
-  // };
-
-  const handleSave = (e: FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    const { firstName, lastName, phone, email, address1, address2, city, state, zip } = e.target as typeof e.target & {
-      firstName: { value: string}, 
-      lastName: { value: string},
-      phone: { value: string}, 
-      email: { value: string}, 
-      address1: { value: string}, 
-      address2: { value: string}, 
-      city: { value: string}, 
-      state: { value: string}, 
-      zip: { value: string}, 
-    };
-
-    const data = {
-      firstName: firstName.value,
-      lastName: lastName.value,
-      phone: phone.value,
-      email: email.value,
-      address1: address1.value,
-      address2: address2.value,
-      city: city.value,
-      state: state.value, 
-      zip: zip.value
-    }
 
-    dispatch(setNewContact(data))
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    dispatch(setNewContact({
```

Comment:
Let me know if this is not what you meant. With my understanding of what you said, I think this is right.